vstruct/primitives.py

```diff
@@ -745,7 +745,7 @@ def vsEmit(self):
 
     def vsGetValue(self):
         cstr = self._vs_value.decode(self._vs_encode, errors='replace')
-        return cstr.split(b'\x00')[0]
+        return cstr.split('\x00')[0]
```

Comment:
is there a unittest for this?  i'm not sure what the right response should be here, so unittest is good ;)

Reply:
There will be after I confirm something works and do a push :smile:

Reply:
:)